framework/src/node/consensus/constants.ts

```diff
@@ -14,6 +14,8 @@
 
 export const EVENT_FORK_DETECTED = 'EVENT_FORK_DETECTED';
 export const EVENT_BLOCK_BROADCAST = 'EVENT_BLOCK_BROADCAST';
+export const EVENT_BLOCK_NEW = 'EVENT_BLOCK_NEW';
```

Comment:
we are prefixing `NETWORK_` or `APP_` to the event constants, maybe we could also categorise rest of them too?